README.rst

```diff
@@ -532,6 +532,12 @@ dependencies, making any newly generated ``requirements.txt`` environment-depend
 As a general rule, it's advised that users should still always execute ``pip-compile``
 on each targeted Python environment to avoid issues.
 
+There is a feature (``--override-environment``) that can be used to
+specify the environment when gathering dependencies, allowing for cross-environment
+fetching. However, a different ``requirements.txt`` must still be generated per
+environment. It is recommended to override all keys in `PEP 508 environment markers`_
+when targetting a different environment so the environment is fully defined.
```

Comment:
It would be great to have a short example in the `README`.

Reply:
Added an example for a 'typical' Linux machine